server/chat/backend/agent/prompt/context_fetchers.py

```
@@ -15,21 +15,25 @@ def build_manual_vm_access_segment(user_id: Optional[str]) -> str:
 
     try:
         from utils.auth.stateless_auth import set_rls_context
+        from utils.secrets.secret_ref_utils import _resolve_org, _org_read_predicate
+        org_id = _resolve_org(user_id)
+        predicate, pred_params = _org_read_predicate(user_id, org_id)
```

Ruff (0.15.12)

[warning] Unpacked variable `predicate` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)